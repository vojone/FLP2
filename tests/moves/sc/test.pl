:- consult('../../../cube').

:- read_cube(C, _), move_s_ccw(C, _, _, RC), write_cube(RC).
