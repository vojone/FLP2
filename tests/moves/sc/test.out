123
122
789
444 111 222 363
111 222 333 454
111 232 333 444
666
344
666
